src/render/terminal.cr

```diff
@@ -82,7 +82,7 @@ module Mint
             when '\n', '\r'
               @cursor = 0
               print char
-            when .whitespace?
+            when .ascii_whitespace?
```

Comment:
can you explain? what's the impact?

Reply:
`Char#whitespace?` includes also unicode characters, which in most of the cases we won't care about.

Reply:
So we can't have emoji? Or we're closing to Chinese community? Because of micro optimization?

Reply:
It's just about whitespace right? Unicode whitespace characters specifically. Also this is just a method which prints stuff the console in a specific width.

Reply:
@gdotdesign Exactly! @Namek seemed to missed the point about what are we talking about here 🤣